go/vt/sqlparser/sql.y

```diff
@@ -602,19 +615,56 @@ base_select:
   }
 
 base_select_no_cte:
-  SELECT comment_opt cache_opt distinct_opt sql_calc_found_rows_opt straight_join_opt select_expression_list where_expression_opt group_by_opt having_opt window_opt
+  SELECT comment_opt cache_opt distinct_opt sql_calc_found_rows_opt straight_join_opt select_expression_list into_opt from_opt where_expression_opt group_by_opt having_opt window_opt into_opt
```

Comment:
You should be able to drop the `into_opt` at the end of this rule, since it's already included at the end of `select_statement` (which will pull this in through `base_select` -> `base_select_no_cte`.